config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
neat, but not anything we need.

Reply:
wanna throw an issue on the backlog to delete these for-the-record-only files, or just a fast-follow PR?